package/MDAnalysis/analysis/msd.py

```diff
@@ -261,16 +262,19 @@ class EinsteinMSD(AnalysisBase):
     ----------
     dim_fac : int
         Dimensionality :math:`d` of the MSD.
-    timeseries : :class:`numpy.ndarray`
+    results.timeseries : :class:`numpy.ndarray`
         The averaged MSD over all the particles with respect to lag-time.
-    msds_by_particle : :class:`numpy.ndarray`
+    results.msds_by_particle : :class:`numpy.ndarray`
         The MSD of each individual particle with respect to lag-time.
     ag : :class:`AtomGroup`
         The :class:`AtomGroup` resulting from your selection
     n_frames : int
         Number of frames included in the analysis.
     n_particles : int
         Number of particles MSD was calculated over.
+
+
+    .. versionadded:: 2.0.0
```

Comment:
I'm correct in thinking this right? (I don't see any MSD entries in 1.1.1).

Reply:
correct